docs/05_operations/13_project-structure/02_settings-plugin.mdx

```diff
@@ -0,0 +1,202 @@
+
+The syntax is similar to Gradle dependencies but not exactly the same. Specifically, you specify the group id, product name and version: `"{groupId}:{productName}:{version}"`
+
+Here is an example of adding a dependency on Auth and Genesis Notify:
+
+```kotlin
+genesis {
+    dependencies {
+        dependency("global.genesis:auth:${extra.properties["authVersion"]}")
+        dependency("global.genesis:genesis-notify:${extra.properties["notifyVersion"]}")
+    }
+}
+```
+
+### Plugins
+
+Other Gradle plugins that are necessary for your project are automatically added by the Settings plugin. Below you can see the plugins that are added for both Legacy and Simplified structures. These plugins can also be disabled if needed.
+
+<Tabs defaultValue="Simplified Structure" values={[{ label: 'Simplified Structure', value: 'Simplified Structure', },{ label: 'Legacy Structure', value: 'Legacy Structure'}]}>
+<TabItem value="Simplified Structure">
+
+| Plugin Name | Plugin ID | Description | Enabled by default |
+| --- | --- | --- | --- |
+| Kotlin | org.jetbrains.kotlin.jvm | Kotlin Gradle plugin | Yes |
+| Genesis Build | global.genesis.build | Core Genesis plugin to enable build tasks, including code generation | Yes |
+| Genesis Distribution | global.genesis.distribution | Creates the project's server distribution zip file. At the moment it does this based on default configuration within the plugin. If you want to customise your distribution then you will need to disable this plugin and create your own configuration using the Gradle distribution plugin. In the future we will support customisation from this plugin directly. | Yes |
+| Genesis Publication Disabler | global.genesis.publication-disabler | Used to disable dictionary-cache sub-module publishing tasks if the project is of type PBC | Yes |
+| Genesis Exec | global.genesis.exec | Creates Gradle tasks used by the Genesis IntelliJ plugin | Yes |
+| Genesis Deploy | global.genesis.deploy | Creates deployment gradle tasks (Note: not required if using Genesis IntelliJ plugin for local development) | No |
+
+</TabItem>
+<TabItem value="Legacy Structure">
+
+| Plugin Name | Plugin ID | Description | Enabled by default |
+| --- | --- | --- | --- |
+| Kotlin | org.jetbrains.kotlin.jvm | Kotlin Gradle plugin | Yes |
+| Genesis Build | global.genesis.build | Core Genesis plugin to enable build tasks, including code generation | Yes |
+| Genesis Distribution | global.genesis.distribution | Creates the project's server distribution zip file. At the moment it does this based on default configuration within the plugin. If you want to customise your distribution then you will need to disable this plugin and create your own configuration using the Gradle distribution plugin. In the future we will support customisation from this plugin directly. | Yes |
+| Genesis Exec | global.genesis.exec | Creates Gradle tasks used by the Genesis IntelliJ plugin | Yes |
```

Comment:
I am not sure whether to mention about exec plugin yet! as all its functionality is exposed through intellij plugin

Reply:
Hmm fair point, but as it is exposed via the plugins block in the Settings plugin, I think we need to mention it for consistency. We could hide it from the plugins block as an alternative.